tests/integration/test_validate_attribute.py

```diff
@@ -73,3 +73,13 @@ def test_url_validation_invalid_url(self, dmge: DataModelGraphExplorer) -> None:
             ],
             [],
         )
+
+    def test__get_target_manifest_dataframes(
+        self, dmge: DataModelGraphExplorer
+    ) -> None:
+        """Testing for ValidateAttribute._get_target_manifest_dataframes"""
```

Comment:
This docstring doesn't add much. Could you add what is special about this test specifically? Or why the specific inputs are being used over another?